lib/grape/validations/params_scope.rb

```diff
@@ -190,7 +190,10 @@ def push_declared_params(attrs, **opts)
       #
       # @return [Array<Symbol>] the nesting/path of the current parameter scope
       def full_path
-        nested? ? @parent.full_path + [@element] : []
+        return (@parent.full_path + [@element]) if nested?
```

Comment:
I dislike the double return, can we do 

```
if
…
elsif 
…
else
…
```

then we can omit the returns altogether

Reply:
Actually, it was my first implementation (with "if-else"), but then rubocop shamed me for huge amount of lines of class.

Reply:
fixed [here](https://github.com/ruby-grape/grape/compare/58d54c34e2c33473063eca40cb9bbd3ebb0b4472..cc54313ef17063d19453b5c8ee052536b83f116c#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL11)

Reply:
I usually just do `rubocop -a ; rubocop --auto-gen-config`.